website/docs/r/record.html.markdown

```diff
@@ -193,6 +208,14 @@ import to ensure that everything is properly escaped and evaluated.
 See [NS1 API](https://ns1.com/api#get-available-metadata-fields) for the most
 up-to-date list of available `meta` fields.
 
+
+#### FQDN Formatting
+Different providers may have different requirements for FQDN formatting.
+A common thing is to return or require a trailing dot, e.g. foo.com.
+The NS1 provider does not require or support trailing or leading dots, 
```

Comment:
technically I think the API will accept a trailing dot (you can check) but it's normalized on the way out. pedantic but maybe better to say "require or return" again.

Reply:
idk support might be better

Reply:
Yeah, I specifically worded it as "provider" since it's the provider and not the API that does the normalization